runtime/onert/core/include/ir/TypeInfo.h

```diff
@@ -52,7 +52,6 @@ class TypeInfo
   DataType type() const { return _type; }
   float scale() const
   {
-    assert(_quant.scales.size() == 1);
```

Comment:
It is not valid for per-channel quantized weight like #9611.
```
#2  0x00007ffff69603fa in __assert_fail_base (fmt=0x7ffff6ae76c0 "%s%s%s:%u: %s%sAssertion `%s' failed.\n%n", assertion=assertion@entry=0x7ffff6434311 "_quant.scales.size() == 1", 
    file=file@entry=0x7ffff64342d8 "/home/gyu/z/ONE/runtime/onert/core/include/ir/TypeInfo.h", line=line@entry=55, 
    function=function@entry=0x7ffff64342a8 "float onert::ir::TypeInfo::scale() const") at assert.c:92
#3  0x00007ffff6960472 in __GI___assert_fail (assertion=0x7ffff6434311 "_quant.scales.size() == 1", file=0x7ffff64342d8 "/home/gyu/z/ONE/runtime/onert/core/include/ir/TypeInfo.h", line=55, 
    function=0x7ffff64342a8 "float onert::ir::TypeInfo::scale() const") at assert.c:101
#4  0x00007ffff60fc285 in onert::ir::TypeInfo::scale (this=0x555555c385f0) at /home/gyu/z/ONE/runtime/onert/core/include/ir/TypeInfo.h:55
#5  0x00007ffff60fc3dc in onert::backend::IPortableTensor::data_scale (this=0x555555c385d0) at /home/gyu/z/ONE/runtime/onert/core/include/backend/IPortableTensor.h:46
#6  0x00007ffff49bfed1 in onert::backend::cpu::ops::GetQuantizedConvolutionMultiplier (input=0x555555c3cc70, filter=0x555555c385d0, bias=0x555555c38510, output=0x555555c3d460, 
    multiplier=0x7fffffffd2e0) at /home/gyu/z/ONE/runtime/onert/backend/cpu/ops/OperationUtils.cc:87
#7  0x00007ffff485d956 in onert::backend::cpu::ops::ConvolutionLayer::convQuant8 (this=0x555555c3b390) at /home/gyu/z/ONE/runtime/onert/backend/cpu/ops/ConvolutionLayer.cc:76
#8  0x00007ffff485e435 in onert::backend::cpu::ops::ConvolutionLayer::run (this=0x555555c3b390) at /home/gyu/z/ONE/runtime/onert/backend/cpu/ops/ConvolutionLayer.cc:192
#9  0x00007ffff624d5fe in onert::exec::FunctionSequence::run (this=0x555555c1fe90) at /home/gyu/z/ONE/runtime/onert/core/src/exec/FunctionSequence.cc:58
#10 0x00007ffff625069a in onert::exec::LinearExecutor::executeImpl (this=0x555555c3d790) at /home/gyu/z/ONE/runtime/onert/core/src/exec/LinearExecutor.cc:50
#11 0x00007ffff624bc53 in onert::exec::ExecutorBase::execute (this=0x555555c3d790, desc=...) at /home/gyu/z/ONE/runtime/onert/core/src/exec/ExecutorBase.cc:139
#12 0x00007ffff62380e6 in onert::exec::Execution::execute (this=0x555555c0bb40) at /home/gyu/z/ONE/runtime/onert/core/src/exec/Execution.cc:211
#13 0x00007ffff7ab3b3a in nnfw_session::run (this=0x555555c34b70) at /home/gyu/z/ONE/runtime/onert/api/src/nnfw_api_internal.cc:510
#14 0x00007ffff7ab0817 in nnfw_run (session=0x555555c34b70) at /home/gyu/z/ONE/runtime/onert/api/src/nnfw_api.cc:111
#15 0x00005555555e19c5 in GenModelTest::TearDown (this=0x555555c34c90) at /home/gyu/z/ONE/tests/nnfw_api/src/GenModelTest.h:390
```

Reply:
`zero_point` uses `zero_points()` for per-channel quantize.